manifests/base/secrets.env

```diff
@@ -1,6 +1,6 @@
 BERT_E_CLIENT_SECRET=vault:secret/data/bert-e#BERT_E_GITHUB_OAUTH_CLIENT_SECRET
 BERT_E_CLIENT_ID=vault:secret/data/bert-e#BERT_E_GITHUB_OAUTH_CLIENT_ID
-BERT_E_GITHOST_PWD=vault:secret/data/bert-e#BERT_E_GITHUB_TOKEN
+BERT_E_ROBOT_PASSWORD=vault:secret/data/bert-e#BERT_E_GITHUB_TOKEN
```

Comment:
Check which version of the repo is specified in the our deployment to ensure that we won't break current installation of bert-e by merging this code.

Reply:
checked, all good, the code is targeting `development/3.7`